tests/tests/fv3_ccpp_sas

```diff
@@ -0,0 +1,71 @@
+###############################################################################
+#
+#  FV3 CCPP SAS test
```

Comment:
What is the tested based? GFSv15.2?

Reply:
It uses the ccpp_gfdlmp.nml.IN namelist template, i.e. it is basically the fv3_ccpp_gfdlmp test with SAS instead of SAMF (both shallow and deep). Same for the IPD version.